datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -1225,12 +1228,33 @@ private void handleLeaderDoAssignment(boolean cleanUpOrphanNodes) {
         _adapter.updateAllAssignments(newAssignmentsByInstance);
       }
 
+      Set<String> failedStreams = Collections.emptySet();
+      // Poll the zookeeper to ensure that hosts claimed assignment tokens for stopping streams
+      if (_config.getEnableAssignmentTokens() &&
+          !PollUtils.poll(() -> _adapter.getNumUnclaimedTokensForDatastreams(stoppingDatastreamGroups) == 0,
+            STOP_PROPAGATION_RETRY_MS, _config.getStopPropagationTimeout())) {
```

Comment:
The current timeout in the stop API handler (in DatastreamResources) is also 60 seconds, so I think we should either update that or either update this `getStopPropagationTimeout` (since this is also 60 seconds) otherwise there could be inconsistencies between what's returned by the stop API and the current state of the datastream.

Reply:
Help me understand how these two timeouts being 60 seconds poses a problem. Is the concern here that the timeout in rest.li API will hit earlier than the stop propagation timeout?

Reply:
Discussed offline. We need to increase the timeout in the rest.li DMS service to give the request handler enough time to hear back from the cluster leader. This change will be included in the next PR.